src/benchmark/trulens/benchmark/benchmark_frameworks/experiments/rag_triad_baselines.ipynb

```diff
@@ -23,241 +23,29 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #50.        # auc = f_auc(scores)
</pre><p>why removing auc?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/1559/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
mainly b/c we have F1 score reported already, and AUC is a less meaningful metric for binary classification cases